model/user/user.go

```diff
@@ -172,7 +173,36 @@ func (u *DBUser) IncPatchNumber() (int, error) {
 		return 0, err
 	}
 	return dbUser.PatchNumber, nil
+}
+
+func (u *DBUser) PatchFinishSubscriber() (*event.Subscriber, error) {
```

Comment:
I'm hoping to change the DBUser type into an interface (, and it seems awkward to hang more data off of this type, but I suppose we can coerce to the concrete type in these cases. 

Is there some way we can seperate or minimize ongoing interface changes?